### PR TITLE
Naji videossass

### PR DESCRIPTION
**Pull request by nalnaji**
Reviewer: @DarryQueen @jonathanshum

### CODE REVIEW

File: app/assets/stylesheets/social_media.sass [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* Color override for social buttons. */
+
```
Comment (DarryQueen) [inline app/assets/stylesheets/social_media.sass]: Yo.

**Comment by DarryQueen** [comment]
One ugly linebreak. Other than that, looks beautiful. Once changes made, go ahead and squash.

File: app/assets/stylesheets/social_media.sass [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    color: #fff
+
+.btn-facebook:hover
+  background-color: #3b5998
+
+.btn-google-plus:hover
+  background-color: #dd4b39
+
+.no-padding
+  padding: 0px
```
Comment (jonathanshum) [inline app/assets/stylesheets/social_media.sass]: important

**Comment by DarryQueen** [comment]
LGTM.

**Comment by jonathanshum** [comment]
LGTM

**Pull request merged**